com.sap.cloud.lm.sl.persistence/src/main/java/com/sap/cloud/lm/sl/persistence/services/ProgressMessageService.java

```diff
@@ -170,6 +171,38 @@ public Integer execute(Connection connection) throws SQLException {
         }
     }
 
+    public int removeAllByProcessIds(final List<String> processIds) throws SLException {

```

Comment:
What about the ProcessLogsService.java. I assume we should remove the process logs as well.

Reply:
Yes, we remove both progress messages and process logs for old processes.